packages/react/src/components/ButtonGroup/__stories__/ButtonGroup.stories.js

```diff
@@ -45,7 +45,7 @@ export const Default = () => {
       copy: text(`Button ${i + 1}`, `Button ${i + 1}`),
       renderIcon:
         iconMap[
-          select(`Icon ${i + 1}`, iconOptions, iconOptions['Arrow Right'])
+          select(`Icon ${i + 1}`, iconOptions, iconOptions[0])
```

Comment:
Given `iconOptions` is not an array, wondering if this change meant to be below:

```suggestion
          select(`Icon ${i + 1}`, iconOptions, iconOptions['Default'])
```

Reply:
yeah resolved the conflicts and corrected the above, Thank you @asudoh